src/parsers/maszyna_parser.cpp

```diff
@@ -40,13 +40,13 @@ namespace godot {
     String MaszynaParser::get_line() {
         PackedByteArray subbuf;
         while (!eof_reached()) {
-            int c = get8();
+            const int c = get8();
             if (c == 10 || c == 13) {
                 break;
             }
             subbuf.append(c);
         }
-        return String::utf8((const char *)subbuf.ptr(), subbuf.size());
+        return String::utf8(reinterpret_cast<const char *>(subbuf.ptr()), subbuf.size());
```

Comment:
please try `return subbuf.get_string_from_utf8()`

Reply:
Godot crashes for me for unexpected reason. Please confirm if that would work and commit it then